### PR TITLE
Update motion.update_metadata: add workflow_id handling.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: tests/system/action/motion/test_update_metadata.py [hunk 1]
```diff
@@ -38,6 +38,45 @@ def test_update_correct(self) -> None:
         assert model.get("tag_ids") == []
         assert model.get("attachment_ids") == []
 
+    def test_update_metadata_workflow_id(self) -> None:
+        self.create_model("meeting/2538", {"name": "name_jkPIYjFz"})
+        self.create_model(
+            "motion/111", {"name": "name_srtgb123", "meeting_id": 2538, "state_id": 88}
```
Comment (jsangmeister) [inline tests/system/action/motion/test_update_metadata.py:44]: add recommendation_id here to make sure it is reset

File: tests/system/action/motion/test_update_metadata.py [hunk 2]
```diff
@@ -38,6 +38,45 @@ def test_update_correct(self) -> None:
...
+                    "action": "motion.update_metadata",
+                    "data": [{"id": 111, "workflow_id": 35}],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        model = self.get_model("motion/111")
+        assert model.get("state_id") == 23
+        assert model.get("recommendation_id") is None
+
```
Comment (jsangmeister) [inline tests/system/action/motion/test_update_metadata.py:79]: Please also add a test where the workflows don't differ and recommendation_id is set and make sure that neither recommendation_id nor state_id is changed.

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: tests/system/action/motion/test_update_metadata.py (hunk 1 above)
Comment (reiterl) [reply tests/system/action/motion/test_update_metadata.py]: Done.

File: tests/system/action/motion/test_update_metadata.py (hunk 2 above)
Comment (reiterl) [reply tests/system/action/motion/test_update_metadata.py:86]: Done. Added a test.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**